src/services/vyos-configd

```diff
@@ -267,6 +267,8 @@ if __name__ == '__main__':
     cfg_group = grp.getgrnam(CFG_GROUP)
     os.setgid(cfg_group.gr_gid)
 
+    os.environ['VYOS_CONFIGD'] = 't'
```

Comment:
What does `t` stand for? Or is it only a virtual marker so the `VYOS_CONFIGD` environment variable is populated?

Why not check if vyos-configd service is running?

Reply:
A virtual/dummy marker: 't' for true, whereas 'true' would be misleading, as it is not a boolean value.

Whether or not vyos-configd is running, the script and its imported modules may not be called under vyos-configd --- though this is the design now in 1.5 after your work, it may not be the case if a daemon timeout or non-fatal error.

To untangle the interactions in T6639, an explicit check is preferred.

Reply:
Thanks for the detailed explanation.